pkg/util/system/BUILD

```diff
@@ -9,13 +9,18 @@ load(
 go_library(
     name = "go_default_library",
     srcs = ["system_utils.go"],
+    deps = [
+        "//cmd/kubeadm/app/constants:go_default_library",
+        "//vendor/k8s.io/api/core/v1:go_default_library",
+    ],
 )
 
 go_test(
     name = "go_default_test",
     srcs = ["system_utils_test.go"],
     library = ":go_default_library",
     deps = [
+        "//cmd/kubeadm/app/constants:go_default_library",
```

Comment:
Yeah, won't be allowed.  You'll have to duplicate the constant.

Reply:
Done, thank you.